amtl/am-hashmap.h

```diff
@@ -135,7 +140,8 @@ class HashMap : private AllocPolicy
   }
   template <typename UK>
   bool add(Insert &i, UK &&key) {
-    Entry entry(ke::Forward<UK>(key), V());
+    V value;
+    Entry entry(ke::Forward<UK>(key), ke::Move(value));
```

Comment:
It looks like to avoid uninitialized values here, you might need to add a default constructor to Entry. Or does `V value = V()` work?